include/knowhere/index.h

```diff
@@ -169,6 +169,7 @@ class Index {
         KNOWHERE_CHECK_EXPECTED(Config::FormatAndCheck(*cfg, json_));
         LOG_KNOWHERE_DEBUG_ << "Search config dump: " << json_.dump();
         KNOWHERE_CHECK_EXPECTED(Config::Load(*cfg, json_, knowhere::SEARCH));
+        KNOWHERE_CHECK_EXPECTED(cfg->Customize(json));
```

Comment:
To my understanding. After loading, the config object is loaded, can we use it directly instead the json?

Reply:
I need use the input json to tell whether ef is set by user or by default value

Reply:
Then this will too confusing since we should not put it after loading.

As mentioned, we don't need to set default value for `ef`, then we can check the config here.